airbyte-integrations/connectors/source-paypal-transaction/source_paypal_transaction/schemas/transactions.json

```diff
@@ -274,24 +320,28 @@
             "type": "object",
             "properties": {
               "incentive_type": {
-                "type": ["null", "string"]
+                "type": ["null", "string"],
+                "maxLength": 500
               },
               "incentive_code": {
-                "type": ["null", "string"]
+                "type": ["null", "string"],
+                "maxLength": 200
               },
               "incentive_amount": {
                 "type": "object",
                 "properties": {
                   "currency_code": {
```

Comment:
Please update the currency code and value according to the [documentation.](https://developer.paypal.com/docs/api/transaction-search/v1/#definition-money)